internal/scylla/client.go

```diff
@@ -116,6 +116,15 @@ func (c *Client) newHttpRequest(ctx context.Context, method, path string, reqBod
 			req.URL.RawQuery = q.Encode()
 		}
 	}
+	headers := bytes.NewBuffer([]byte{})
+	_ = req.Header.Write(headers)
```

Comment:
I am not quite sure why you need this, can this be removed?

Reply:
Agreed, we keep nothing interesting on headers